### PR TITLE
T170416 add tests showing case-sensitity of createChecksum

### PR DESCRIPTION
**Pull request by wiese**
master is red with this!

### CODE REVIEW

File: contexts/PaymentContext/tests/Unit/Domain/ChecksumGeneratorTest.php [hunk 1]
```diff
@@ -43,6 +43,19 @@ public function testCanGenerateChecksumWithManyCharacters(): void {
...
+		$generator = new ChecksumGenerator( str_split( 'ACDEFKLMNPRSTWXYZ349' ) );
+
+		$this->assertSame( 'F', $generator->createChecksum( 'aAaa-aaaa-' ) );
+		$this->assertSame( 'F', $generator->createChecksum( 'aaaa-aaaa-' ) );
+		$this->assertSame( '4', $generator->createChecksum( 'abcd-efGh-' ) );
+		$this->assertSame( '4', $generator->createChecksum( 'abcd-efgh-' ) );
+		$this->assertSame( 'K', $generator->createChecksum( 'QaQa-qaqa-' ) );
+		$this->assertSame( 'K', $generator->createChecksum( 'qaqa-qaqa-' ) );
+		$this->assertSame( '9', $generator->createChecksum( 'aaAa-XXXX-' ) );
+		$this->assertSame( '9', $generator->createChecksum( 'aaaaxxxx-' ) );
```
Comment (JeroenDeDauw) [inline contexts/PaymentContext/tests/Unit/Domain/ChecksumGeneratorTest.php]: sneaky refugee test hiding in this method :)

Comment (wiese) [reply contexts/PaymentContext/tests/Unit/Domain/ChecksumGeneratorTest.php]: Well, I warned that the master would be red with this.

Comment (JeroenDeDauw) [reply contexts/PaymentContext/tests/Unit/Domain/ChecksumGeneratorTest.php]: I mean that this last line is also testing something else

**Comment by JeroenDeDauw** [comment]
https://github.com/wmde/FundraisingFrontend/pull/966

**Pull request closed without merging**